integration_tests/serverless.yml

```diff
@@ -0,0 +1,84 @@
+service: integration-tests-lambda-extension
+
+resources:
+    Description: "[Serverless] Integration tests application for lambda extension"
+
+provider:
+  name: aws
+  region: sa-east-1
+  tracing:
+    lambda: true
+    apiGateway: true
+  environment:
+    DD_INTEGRATION_TEST: true
+    DD_DD_URL: http://127.0.0.1:3333
+    DD_API_KEY: NO_NEED_TO_BE_VALID
+    DD_LOGS_CONFIG_LOGS_DD_URL: 127.0.0.1:3333
+    DD_LOGS_CONFIG_LOGS_NO_SSL: true
+    DD_LOG_LEVEL: INFO
+    DD_LOGS_ENABLED: false
+    DD_SERVERLESS_LOGS_ENABLED: false
+    DD_MERGE_XRAY_TRACES: false
+    DD_TRACE_ENABLED: false
+
+  lambdaHashingVersion: 20201221
+  timeout: 15
+  deploymentBucket:
+    name: integration-tests-deployment-bucket
+  iam:
+    # IAM permissions require that all functions are deployed with this role
+    role: "arn:aws:iam::601427279990:role/serverless-integration-test-lambda-role"
+
+layers:
+  recorderExtension:
+    package:
+      artifact: ./recorder-extension/ext.zip
+  datadogExtensionIntegrationTest:
+    package:
+      artifact: ../.layers/datadog_extension.zip
+
+functions:
+  enhancedMetricTest:
+    name: integration-tests-lambda-extension-${sls:stage}-enhancedMetricTest
+    runtime: nodejs12.x
+    handler: src/metric.enhancedMetricTest
+    layers:
+      - arn:aws:lambda:${self:provider.region}:601427279990:layer:Datadog-Node12-x:${env:LAYER_VERSION}
```

Comment:
I think we should pull the Node layer from the prod account (464622532012). That way we are testing the development version of the extension with the production Lambda Library.

Reply:
✅